sdk/python/kfp/cli/output.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from enum import Enum, unique
+
+from tabulate import tabulate
+
+
+@unique
+class OutputFormat(Enum):
+    table = "table"
+    json = "json"
+
+
+def print_output(data, headers, output_format, table_format='simple'):
+    if output_format == "table":
```

Comment:
Regarding the output_format, I'll recommend to use an enum instead of free-formed string here.

Or throw NotImplementedError if unknown format is encountered.

Reply:
Reading from below `output_format` is actually a pass-through, so string w/ proper error raised looks reasonable to me.

Reply:
I missed this earlier, fixed it now to compare with the enum names.